Dockerfile

```diff
@@ -35,58 +35,26 @@ RUN cat /usr/local/bin/start.sh
 
 USER $NB_USER
 
+RUN mkdir /home/$NB_USER/tmp
+COPY base-env.yml /home/$NB_USER/tmp/base-env.yml
 # Install Python 3 packages
 # Remove pyqt and qt pulled in for matplotlib since we're only ever going to
 # use notebook-friendly backends in these images
 
-RUN conda install -c bokeh --yes \
-    datashader
+RUN conda env update --name root --file /home/$NB_USER/tmp/base-env.yml 
 
-RUN conda install -c conda-forge --yes \
-    altair \
-    basemap \
-    basemap-data-hires \
-    boto \
-    boto3 \
-    bokeh \ 
-    cloudpickle \
-    dill \
-    h5py \ 
-    hdf5 \
-    ipyleaflet \
-    ipywidgets \
-    lxml \
-    matplotlib \ 
-    netcdf4 \
-    networkx \
-    numba \
-    numpy \
-    owslib \
-    obspy \ 
-    pandas \
-    pillow \
-    plotly \
-    requests \
-    scikit-learn \
-    seaborn \
-    streamz \
-    vega \
-    vega_datasets \
-    xarray && \
-    conda remove --quiet --yes --force qt pyqt && \
-    conda clean -tipsy
-
-# PIP Packages
-RUN pip install python-cmr
+RUN rm -rf /home/$NB_USER/tmp
 
-RUN pip install dash==0.20.0 \
-    dash-renderer==0.11.3 \
-    dash-html-components==0.8.0 \
-    dash-core-components==0.18.1
-
-RUN pip install nbgitpuller
+# Install tutorial environments
+RUN git clone https://github.com/oceanhackweek/ohw2018_tutorials.git && \
+    cd ohw2018_tutorials && \
+    conda env update --name root --file day2/ioos_data_access/environment.yml && \
+    conda env update --name root --file day2/ooi_data_access/environment.yml && \
+    conda env update --name root --file day3/geospatial_and_mapping_tools/environment.yml && \
+    conda env update --name root --file day4/data-mining/environment.yml && \
```

Comment:
To continue the `defaults` removal we would need to go into all of those env files and remove it if `defaults` is listed. (I already removed if from mine BTW.)